api/nakadi-event-bus-api.yaml

```diff
@@ -925,6 +1199,67 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Subscriptions allow applications to easily scale the
+      number of clients by managing consumed event offsets and distributing load between instances.
+      The key properties that identify subscription are 'owning_application', 'event_types' and 'consumer_group'.
+      It's not possible to have two different subscriptions with these properties being the same.
+    properties:
+      id:
+        type: string
+        format: uuid
+        readOnly: true
+        description: |
+          Id of subscription that was created. Is generated by Nakadi, should not be specified when creating
+          subscription.
+      owning_application:
+        type: string
+        example: 'gizig'
+        description: |
+          The id of application owning the subscription. The reading and modification of subscription will be limited
+          to this application id.
+      event_types:
+        type: array
+        items:
+          type: string
+        description: |
+          EventTypes to subscribe to.
+          The order is not important. Subscriptions that differ only be the order of EventTypes will be
+          considered the same and will have the same id.
+
+          * Currently only subscription to a single EventType is supported. Subscriptions with more than one
+            EventType in event_types property will be rejected.
```

Comment:
> Do you see some weak points here?

I guess the point is that cardinality changes can be problematic, and we should be pretty sure the params and protocol aspects (eg empty batch keepalives, disconnecting non-updating clients, topic deletion) will work with multiple topics, in particular where those topics have different levels of throughput. We don't have an actual way to look at what happens with real clients and servers right now because it's not implemented, so it needs to be reasoned about as best we can. I'll go through them the way I did with parameters in https://github.com/zalando/nakadi/pull/353#issuecomment-238514240.